app/pages/admin/project-status.jsx

```diff
@@ -124,17 +124,23 @@ class ProjectStatus extends Component {
       .catch(error => this.setState({ error }));
   }
 
+  saveProject(project) {
+    return project.save()
+      .catch(error => this.setState({ error }));
+  }
+
   handleFeaturedProjectChange({ target }) {
-    const { featured, project } = this.state;
-    if (featured) {
-      featured.update({ featured: false });
-      featured.save()
-        .then(() => this.setState({ featured }))
+    const { featuredProject, project } = this.state;
+    project.update({ featured: target.checked });
+    if (featuredProject) {
```

Comment:
This will be true even if `featuredProject.featured` is false.

Reply:
Yeas, but I'm checking if a featured project exists at all.

Reply:
In that case, you will update the featured project even when `featuredProject.featured` is false.

Reply:
I don't think so, because [`getFeaturedProject`](https://github.com/zooniverse/Panoptes-Front-End/blob/4c8da67db77815b9ce7900a80dbf5fbb093a1be5/app/pages/admin/project-status.jsx#L60) add a project to local state only if `featured: true`.